3761/1/top_gems.rb

```diff
@@ -0,0 +1,13 @@
+require 'mechanize'
+require 'uri'
+require './argv_parser.rb'
+require './gem_list_creator.rb'
+require './gem_statistic_fetcher.rb'
+require './gem_statistic_analyzer.rb'
+require './table_creator.rb'
+require './top_gem.rb'
+require './page_fetcher.rb'
+FILE_NAME_DEFAULT = '.top-gems.yml'.freeze
```

Comment:
Missing magic comment `# frozen_string_literal: true.` Better then freeze

Reply:
it is rubocop error(